CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ pull request if there was one.
 
 ### Added:
 
+- [Enable Druid In-filter in Fili](https://github.com/yahoo/fili/pull/688)
+    * `DefaultDruidFilterBuilder` is renamed to `DruidOrFilterBuilder`
+    * Implement `DruidInFilterBuilder` which turns list of selector filter generated by `DruidOrFilterBuilder` into a
+      Druid in-filter. The in-filter could substantially shorten druid query, making the query more sharable and
+      readable.
```

Comment:
Why is "sharable and readable" desired? Since the Druid query is system-facing (ie. it intended for consumption by Druid, not a human), what impact does an in-filter have on that?

Reply:
Debugging mostly. If there's a bug (or you're trying to do some analysis of the 
logs for whatever reason), and it's not clear if it's in Druid or
the API, then you may need to send your resident Druid expert the Druid query
that's being sent down.

Reply:
Fair enough. Does it make sense to add that bit of "why do we care about this?" to the change log?